Added February Community Call (#1065)
USRSE/usrse.github.io: _events/2022/2023-02-community-call.md

---
title: February 2023 Community Call
expires: 2023-02-11
event_date: "February 10, 2023"
layout: event
duration: 60
repeated: false
category: community-call
time:
    - - start: 2023-02-10T19:00:00Z
        end: 2023-02-10T20:00:00Z
---

The February community call will be about **Workflow Tools** on **February 10, 2pm ET/1pm CT/12pm MT/11am PT**.

> Workflow tools figure prominently in some RSE fields and are barely used in others. In this call we will compare notes on how people are working with workflow tools, what tools are working well and when, what are up and coming practices to learn, and how the [workflows community](https://workflows.community) has organized themselves to tackle the grand workflows challenges. Everyone is welcome as always. Whether you are a passionate user of widely-used workflow systems (e.g., Nextflow, Parsl, Snakemake, CWL, etc.) or philosophically wedded to nothing but bash it would be great to hear perspectives and exchange ideas. 

Reviewer: ```suggestion
Workflow tools figure prominently in some RSE fields and are barely used in others. In this call we will compare notes on how people are working with workflow tools, what tools are working well and when, what are up and coming practices to learn, and how the [workflows community](https://workflows.community) has organized themselves to tackle the grand workflows challenges. Everyone is welcome as always. Whether you are a passionate user of widely-used workflow systems (e.g., Nextflow, Parsl, Snakemake, CWL, etc.) or philosophically wedded to nothing but `bash`, we invite you to join to hear perspectives and exchange ideas. 
```
